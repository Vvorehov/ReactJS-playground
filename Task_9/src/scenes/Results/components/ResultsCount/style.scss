.results-count {
  flex-grow: 1;
  font-size: 1.4rem;
  font-weight: 700;
  padding: 0 2rem;
}