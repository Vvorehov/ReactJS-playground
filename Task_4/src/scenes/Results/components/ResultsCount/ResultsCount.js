import React, {Component} from 'react';
import { render } from "react-dom";
import './style.scss'

class ResultsCount extends Component {
  render() {
    return (
      <div className="results-count"><span id="results-count">{ this.props.items }</span> movies found</div>
    )
  }
}

export default ResultsCount;