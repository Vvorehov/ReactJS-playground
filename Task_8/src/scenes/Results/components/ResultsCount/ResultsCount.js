import React, {Component} from 'react';
import { render } from "react-dom";
import './style.scss'

class ResultCount extends Component {
  render() {
    return (
      <div className="results-count"><span id="results-count">{ this.props.items }</span> movies found</div>
    )
  }
}

export default ResultCount;