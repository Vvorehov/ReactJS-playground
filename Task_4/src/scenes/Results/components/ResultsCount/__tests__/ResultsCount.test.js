import React from "react";
import ResultsCount from "../ResultsCount";
import TestRenderer from 'react-test-renderer';

describe('ResultsCount layout complete', () => {
    test('Render correct', () => {

        const component = TestRenderer.create(
            <ResultsCount items="7" />
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

