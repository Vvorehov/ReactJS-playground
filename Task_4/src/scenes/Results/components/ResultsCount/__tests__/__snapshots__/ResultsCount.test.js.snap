// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsCount layout complete Render correct 1`] = `
<div
  className="results-count"
>
  <span
    id="results-count"
  >
    7
  </span>
   movies found
</div>
`;
